test.sh

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 # Exit on first error and echo commands
-set -e
-set -x
+set -ex
```

Comment:
I prefer to keep my option setting on separate lines. Please revert this change.